.vscode/settings.json

```diff
@@ -10,5 +10,6 @@
   "grunt.autoDetect": "off",
   "javascript.validate.enable": false,
   "npm.packageManager": "yarn",
-  "typescript.validate.enable": true
+  "typescript.validate.enable": true,
+  "testing.openTesting": "neverOpen"
```

Comment:
Does this need to be saved as a universal setting, or is this more of a personal preference?